application/views/questionAdministration/summary.php

```diff
@@ -211,7 +211,7 @@
                 <tr>
                     <td>
                         <strong>
-                            <?php echo $setting['caption'];?>:
+                            <?php eT($setting['caption']);?>:
```

Comment:
This will only work if setting[caption] already is wrapped around gT or eT somewhere else. Is it?

Reply:
Yes, should be already there. $setting['caption'] holds the name of the attribute.
If the literal is not anywhere else in the code and wasn't added manually, we can add it to a special fiel created for holding literals. (and allowing the translation bot to catch them)

Reply:
Caption in question helper is already wrapped in gT(). So eT() shouldn't be needed here...?

```
        "caption" => gT('Sort answers alphabetically'));
```

Reply:
And description in QuestionType.php (but also saved in database???):

```
                'description' => gT("Multiple numerical input", "html", $language),
```

Reply:
Let me review

Reply:
Not sure if we are on the same page.
The ticket talks about some translations missing.
Ticket already diagnoses that those aren't available for translating.
But they are. Ex: `Use slider layout` is available for translating.

Still, not showing the translations.
The fix is about showing the translations.

Reply:
OK, that makes sense.